src/Compiler/AbstractIL/il.fs

```diff
@@ -1203,10 +1203,11 @@ type ILAttribute =
 
 [<NoEquality; NoComparison; Struct>]
 type ILAttributes(array: ILAttribute[]) =
+    member _.AsArray() = array
 
-    member x.AsArray() = array
+    member _.AsList() = array |> Array.toList
 
-    member x.AsList() = array |> Array.toList
+    static member internal Empty = ILAttributes([||])
```

Comment:
Note this is a static getter, one of the gotchas in F#  - it reallocated on every invocation

You probably want

```fsharp
    static member val internal Empty = ILAttributes([||])
```

Reply:
Yeah, always forgetting it, thanks.